_episodes/21.md

```diff
@@ -0,0 +1,211 @@
+statements. dbt won't execute the SQL itself, rather it pushes all of the 
+compute down to the SQL engine. And this SQL engine can be Trino, or an engine 
+included in the data source like the database itself.
+
+### Transformation vs ad-hoc joins
+
+Transformations done by dbt are in general used to clean and prepare data for 
+analytics purposes. It's often used to go from the raw data to a ready-to-use 
+data for reporting and analysis. dbt will create database objects like tables 
+or views to be consumed by business users and analytics tools.
+
+On the other hand, even if Trino can also execute SQL to create tables and 
+views, these SQL queries are not managed but just executed. Trino doesn't have,
+like dbt, all the framework to version, audit, document and orchestrate SQL 
+script and execution. In general Trino is more used to execute SQL SELECT 
+statements generated by users or BI tools to analyze data in an interactive way.
+
+### Cases for why you need both
+
+Both solutions are complementary when you need to access different sources from
+a single SQL query or when you need to run SQL query with good performance on
+object storages (like S3, GCS, ADLS or HDFS).
+
+It's where Trino can complement dbt, as dbt can only access a single data 
+warehouse connection in a SQL query. In dbt there is no way to query multiple 
+storage systems at the same time.
+
+Also, for object storage (data lake), Trino is recognized to have great 
+performance, so it's a good way, with dbt, to transform and prepare data at 
+large scale in object storage. Trino also allows you to run dbt on a 
+traditional, on-premise data warehouse where normally dbt only runs on a modern
+cloud data warehouse like Snowflake, BigQuery, or Redshift.
+
+### dbt basics
+    
+A good tutorial can be [found here](https://docs.getdbt.com/tutorial/setting-up)
+which covers the fundamental topics of dbt for you to learn:
+* Project : A directory of SQL and YAML files defined with a single project file.
+* Models: A model is a single SQL file where you define your transformations to create a table or a view.
+* Profile : To define connections to your data sources.
+* Then you have other resources like seeds, macros, tests, sources, snapshots.
+
+## Demo: Contributing to Trino
+
+Victor will now run the demo from 
+[the blog he wrote that inspired this episode](https://medium.com/geekculture/trino-dbt-a-match-in-sql-heaven-1df2a3d12b5e).
+
+If you looked at the code, you  may have noticed that the code used an adapter 
+called `db-presto-trino`. This adapter derives from the outdated presto naming and is
+still there for interaction with legacy Presto clusters. Although it can work
+it uses an outdated python client to interact with Trino and there is an open
+[issue to create an official `dbt-trino` adapter](https://github.com/dbt-labs/dbt-presto/issues/39) 
+that uses the updated [trino-python-client](https://github.com/trinodb/trino-python-client).
```

Comment:
We should add an update that a new client now exists in the call below..  or after.